test/room_test.exs

```diff
@@ -39,9 +39,55 @@ defmodule RoomTest do
     wait_for_breaks(2)
   end
 
-  test "send to peer" do
+  test "join room with turn" do
+    System.put_env("SIGNALTOWER_TURN_SECRET", "verysecretpassphrase1234")
     room_pid = create_room("r-room3")
 
+    go = fn timestamp_before ->
+      spawn_user_no_join(fn ->
+        GenServer.call(room_pid, {:join, self(), %{user: "1"}, 0})
+
+        assert_receive(
+          {:to_user,
+           %{
+             event: "joined_room",
+             own_id: own_id,
+             turn_user: user,
+             turn_password: pw
+           }},
+          1000
+        )
+
+        [timestamp_str, id] = String.split(user, ":")
+        {timestamp, ""} = Integer.parse(timestamp_str)
+        assert own_id == id
+        assert System.os_time(:second) < timestamp
+        assert timestamp < System.os_time(:second) + 3 * 60 * 60 + 10
```

Comment:
I hope this line will not get flaky in CI

Reply:
I hope that too :D

Reply:
For my understanding. The clock is faked in this test right?

Reply:
You mean System.os_time(:second)? No, that's not faked